include/dlaf/sender/traits.h

```diff
@@ -27,7 +30,55 @@ struct SenderSingleValueTypeImpl<TypeList<TypeList<T>>> {
   using type = T;
 };
 
-struct empty_env {};
+// Utilities to use unique_t
```

Comment:
Leave out or update to `UniqueType`? The utilities may be useful on their own at some point so my preference would be to leave it out.